src/routes/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const router = express.Router();
+
+router.get('/', function (req, res) {
+    res.send(`<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="theme-color" content="#000000">
```

Comment:
What does the "theme-color" meta tag do?

Reply:
Controls the color of the toolbar on Android. See here: https://developers.google.com/web/updates/2014/11/Support-for-theme-color-in-Chrome-39-for-Android

Came by default w/ create-react-app.